src/controllers/activegate/reconciler/statefulset/statefulset.go

```diff
@@ -27,12 +28,25 @@ const (
 	DTIdSeedClusterId    = "DT_ID_SEED_K8S_CLUSTER_ID"
 	DTNetworkZone        = "DT_NETWORK_ZONE"
 	DTGroup              = "DT_GROUP"
-	DTInternalProxy      = "DT_INTERNAL_PROXY"
 	DTDeploymentMetadata = "DT_DEPLOYMENT_METADATA"
 
-	ProxySecretKey = "proxy"
-
 	ActivegateContainerName = "activegate"
+
+	InternalProxySecretMountPath = "/var/lib/dynatrace/secrets/internal-proxy"
+
+	InternalProxySecretVolumeName = "internal-proxy-secret-volume"
+
+	InternalProxySecretHost          = "host"
+	InternalProxySecretHostMountPath = InternalProxySecretMountPath + "/" + InternalProxySecretHost
+
+	InternalProxySecretPort          = "port"
+	InternalProxySecretPortMountPath = InternalProxySecretMountPath + "/" + InternalProxySecretPort
+
+	InternalProxySecretUsername          = "username"
+	InternalProxySecretUsernameMountPath = InternalProxySecretMountPath + "/" + InternalProxySecretUsername
+
+	InternalProxySecretPassword          = "password"
+	InternalProxySecretPasswordMountPath = InternalProxySecretMountPath + "/" + InternalProxySecretPassword
```

Comment:
small nitpick: A noice amount of `const`s here, maybe moving them to `src/controllers/activegate/reconciler/statefulset/config.go` would make stuff look cleaner ?
WDYT?

Reply:
Good idea